artifactory/spec/specfiles.go

```diff
@@ -228,8 +229,8 @@ func ValidateSpec(files []File, isTargetMandatory, isSearchBasedSpec, isUpload b
 		if isRegexp && isAnt {
 			return errors.New("Can not use the option of regexp and ant together.")
 		}
-		if isArchive && isSymlinks {
-			return errors.New("Symlinks cannot be stored in an archive.")
+		if isArchive && isSymlinks && isExplode {
+			return errors.New("Symlinks cannot be stored in an archive that will be exploded in artifactory.\nWhen uploading symlink to artifactory a file in size 0 with properties describing the symlink is being saved.\nThis process is not yet supported by artifactory when exploding symlinks from a zip.\nPossible uses: archive with symlinks or archive with explode.")
```

Comment:
Symlinks cannot be stored in an archive that will be exploded in artifactory.\nWhen uploading symlink to artifactory a file in size 0 with properties describing the symlink is being saved.\nThis process is not yet supported by artifactory when exploding symlinks from a zip.\nPossible uses: archive with symlinks or archive with explode.

-->

Symlinks cannot be stored in an archive that will be exploded in artifactory.\nWhen uploading a symlink to Artifactory, the symlink is represented in Artifactory as 0 size filewith properties describing the symlink.\nThis symlink representation is not yet supported by Artifactory when exploding symlinks from a zip.)